nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/repository/StandardProcessSession.java

```diff
@@ -175,6 +184,10 @@ public StandardProcessSession(final ProcessContext context) {
         this.sessionId = idGenerator.getAndIncrement();
         this.connectableDescription = description;
 
+        this.isRollbackCountEnabled = NiFiProperties.getInstance().isRollbackCountEnabled();
+        this.isRollbackLogUnackFFEnabled = NiFiProperties.getInstance().isRollbackLogUnackFFEnabled();
+        this.rollbackLogUnackFFMax = NiFiProperties.getInstance().getRollbackLogUnackFFMax();
+
```

Comment:
Yeah...It just feels like attributes aren't the right place for this.  Will reply more thoughtfully on the JIRA itself.